Makefile

```diff
@@ -1038,6 +1052,9 @@ clean: clean-tests clean-object_creator clean-pch clean-lang
 	rm -rf *$(BINDIST_DIR) *cataclysmdda-*.tar.gz *cataclysmdda-*.zip
 	rm -f $(SRC_DIR)/version.h $(SRC_DIR)/prefix.h
 	rm -f $(CHKJSON_BIN)
+	rm -f src/third-party/imtui/*.o
+	rm -f src/third-party/imgui/*.o
```

Comment:
The reason we nuke the whole object directory (and essentially the reason we have a separate obj directory) is that the compiler emits multiple intermediate files, not just .o, so those all need to be cleaned up (and I recommend a separate obj directory).

Reply:
rewrote Makefile changes, this statement is no longer necessary, fixed.